# 
# Submitted-By: Robert Schwebel, 2006-01-02
# Committed-By: Robert Schwebel
#
# Error: 
#
# scaffold.c:28: error: conflicting types for 'malloc'
# scaffold.c:28: error: conflicting types for 'malloc'
#
# Description: 
#
# Patch taken from LFS:
# http://www.linuxfromscratch.org/patches/downloads/tcp_wrappers/tcp_wrappers-7.6-gcc34-1.patch
#
# State:
# 
# unknown 
#

diff -aur tcp_wrappers_7.6/scaffold.c tcp_wrappers_7.6-new/scaffold.c
--- tcp_wrappers_7.6/scaffold.c	1997-03-21 10:27:24.000000000 -0800
+++ tcp_wrappers_7.6-new/scaffold.c	2004-04-10 14:11:01.653193120 -0700
@@ -25,7 +25,6 @@
 #define	INADDR_NONE	(-1)		/* XXX should be 0xffffffff */
 #endif
 
-extern char *malloc();
 
 /* Application-specific. */
